PsfLauncher/main.cpp

```diff
@@ -268,43 +277,233 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
         else
         {
-            //Log("error starting monitor using SellExecuteEx also. Error=0x%x\n", ::GetLastError());
+            std::wostringstream ss;
+            ss << L"error starting monitor using SellExecuteEx";

```

Comment:
no need for string stream

Reply:
Good catch.  I went ahead and turned all but one instance of ss into a std::wstring.